frontend/src/component/feature/FeatureView/FeatureOverview/ReleasePlan/ReleasePlan.tsx

```diff
@@ -187,6 +225,27 @@ export const ReleasePlan = ({
                 onConfirm={onRemoveConfirm}
                 environmentActive={!environmentIsDisabled}
             />
+            <ReleasePlanAddChangeRequestDialog
+                action='deleteReleasePlan'
+                environmentId={environment}
+                featureId={featureName}
+                onClosing={() => setPlanForChangeRequestRemoveDialog(undefined)}
+                projectId={projectId}
+                releasePlan={planForChangeRequestRemoveDialog}
+                environmentActive={!environmentIsDisabled}
+            />
+            <ReleasePlanAddChangeRequestDialog
+                action='startMilestone'
+                environmentId={environment}
+                featureId={featureName}
+                onClosing={() => {
+                    setPlanForChangeRequestStartMilestoneDialog(undefined);
+                    setMilestoneForChangeRequestDialog(undefined);
+                }}
+                projectId={projectId}
+                releasePlan={planForChangeRequestStartMilestoneDialog}
+                milestone={milestoneForChangeRequestDialog}
+            />
```

Comment:
I set two up here so I wouldn't have to let the action be nullable & have another react state for it. I could change the logic in the dialog to looking at milestone being set, only release plan being set, or only template being set though. WDYT @nunogois?

Reply:
Uh sure. Honestly I would have expected separate dialog components for each action, but if this works and you're happy I don't think it matters much 🤷